README.md

```diff
@@ -1,93 +1,82 @@
-2. Quantize the weights and activations: works best for compute bound models
-2. Quantize the activations and weights and sparsify the weight
+[View detailed benchmarks](torchao/quantization/README.md) |  [Learn about sparsity](torchao/sparsity/README.md)
 
-For gpt-fast `Int4WeightOnlyConfig()` is the best option at bs=1 as it **2x the tok/s and reduces the VRAM requirements by about 65%** over a torch.compiled baseline.
+### Getting Started with Quantization
 
-If you don't have enough VRAM to quantize your entire model on GPU and you find CPU quantization to be too slow then you can use the device argument like so `quantize_(model, Int8WeightOnlyConfig(), device="cuda")` which will send and quantize each layer individually to your GPU.
+Quantize any model with `nn.Linear` layers (including HuggingFace models) in just one line:
 
-If you see slowdowns with any of these techniques or you're unsure which option to use, consider using [autoquant](./torchao/quantization/README.md#autoquantization) which will automatically profile layers and pick the best way to quantize each layer.
+#### Option 1: Direct TorchAO API
 
 ```python
-model = torchao.autoquant(torch.compile(model, mode='max-autotune'))
+from torchao.quantization.quant_api import quantize_, Int4WeightOnlyConfig
+quantize_(model, Int4WeightOnlyConfig())
 ```
 
-We also provide a developer facing API so you can implement your own quantization algorithms so please use the excellent [HQQ](https://github.com/pytorch/ao/tree/main/torchao/prototype/hqq) algorithm as a motivating example.
+#### Option 2: HuggingFace Integration
 
-### Evaluation
+```python
+from transformers import TorchAoConfig, AutoModelForCausalLM
+from torchao.quantization.quant_api import Int8WeightOnlyConfig
+
+# Create quantization configuration
+quantization_config = TorchAoConfig(quant_type=Int8WeightOnlyConfig())
+
+# Load and automatically quantize
+quantized_model = AutoModelForCausalLM.from_pretrained(
+    "meta-llama/Llama-3.1-8B-Instruct",
+    torch_dtype="auto",
+    device_map="auto",
+    quantization_config=quantization_config
+)
+```
 
-You can also use the EleutherAI [LM evaluation harness](https://github.com/EleutherAI/lm-evaluation-harness) to directly evaluate models
-quantized with post training quantization, by following these steps:
+### Deployment with vLLM
 
-1. Quantize your model with a [post training quantization strategy](#post-training-quantization).
-2. Save your model to disk or upload to huggingface hub ([instructions]( https://huggingface.co/docs/transformers/main/en/quantization/torchao?torchao=manual#serialization)).
-3. [Install](https://github.com/EleutherAI/lm-evaluation-harness?tab=readme-ov-file#install) lm-eval.
-4. Run an evaluation. Example:
+Deploy quantized models with one command:
 
-```bash
-lm_eval --model hf --model_args pretrained=${HF_USER}/${MODEL_ID} --tasks hellaswag --device cuda:0 --batch_size 8
+```shell
```

Comment:
feels a bit strange to have the HF command for llama3 and this one for Phi-4

Reply:
agreed,  will make it consistent